config/locales/forms/2023/sales/property.en.yml

```diff
@@ -0,0 +1,105 @@
+                        check_answer_label: "UPRN"
+                        hint_text: ""
+                        question_text: "What is the property's UPRN?"
+
+                    uprn_known:
+                        page_header: ""
+                        check_answer_label: "UPRN known?"
+                        hint_text: "The Unique Property Reference Number (UPRN) is a unique number system created by Ordnance Survey and used by housing providers and various industries across the UK. An example UPRN is 10010457355.<br><br>The UPRN may not be the same as the property reference assigned by your organisation.<br><br>If you don’t know the UPRN you can enter the address of the property instead on the next screen."
+                        question_text: "Do you know the property's UPRN?"
+
+                    uprn_confirmed:
+                        page_header: ""
+                        check_answer_label: "Is this the right address?"
+                        hint_text: ""
+                        question_text: "Is this the property address?"
+
+                    address:
+                        page_header: "Q15 - What is the property's address?"
+                        address_line1:
+                            check_answer_label: "Address lines 1 and 2"
+                            hint_text: ""
+                            question_text: "Address line 1"
+                        address_line2:
+                            check_answer_label: ""
+                            hint_text: ""
+                            question_text: "Address line 2 (optional)"
+                        town_or_city:
+                            check_answer_label: "Town or city"
+                            hint_text: ""
+                            question_text: "Town or city"
+                        county:
+                            check_answer_label: "County"
+                            hint_text: ""
+                            question_text: "County (optional)"
+                        postcode_full:
+                            check_answer_label: "Postcode"
+                            hint_text: ""
+                            question_text: "Postcode"
+
+                    la:
+                        page_header: ""
+                        check_answer_label: "Local authority"
+                        hint_text: ""
+                        question_text: "What is the property’s local authority?"
+
+                    income1_value_check:
+                        page_header: ""
+                        check_answer_label: "Buyer 1 income confirmation"
+                        hint_text: ""
+                        question_text: "Are you sure this is correct?"
+
+                    income2_value_check":
+                        page_header: ""
+                        check_answer_label: "Buyer 2 income confirmation"
+                        hint_text: ""
+                        question_text: "Are you sure this is correct?"
```

Comment:
`income1_value_check` and `income2_value_check` are also used in the **income, benefits and savings** subsection, will remove from this translation file.

Reply:
and `combined_income_value_check`

Reply:
Also, `percentage_discount_value_check` which is also used in the discounted_ownership_scheme subsection.
And, monthly_charges_value_check used in: outright_sale, discounted_ownership_scheme and shared_ownership_scheme.